atom/browser/api/atom_api_top_level_window.cc

```diff
@@ -780,6 +780,14 @@ v8::Local<v8::Value> TopLevelWindow::GetContentView() const {
     return v8::Local<v8::Value>::New(isolate(), content_view_);
 }
 
+void TopLevelWindow::SetWindowControlsVisibility(bool visible) {
+  window_->SetWindowControlsVisibility(visible);
+}
+
+bool TopLevelWindow::IsWindowControlsVisible() {
```

Comment:
(nit): `IsWindowControlsVisible` may be better as `AreWindowControlsVisible`

Reply:
Fixed in the latest commit